### ISSUE TITLE
"Internal Server Error - A non well formed numeric value encountered" on channel entry pages

### ISSUE THREAD

**Issue by MichaelLotriet** [khalwat/seomatic#191]
On my channel entry pages i am getting a Internal Server Error - A non well formed numeric value encountered.
This started when i updated to Craft CMS 2.6.2960.
As soon as i disable SEOMatic the entries display fine.

**Comment by khalwat** [comment]
What version of SEOmatic are you using?

**Comment by MichaelLotriet** [comment]
I am using version 1.1.40.

**Comment by khalwat** [comment]
Okay. Can you enable `devMode` and get me the debug output?

**Comment by MichaelLotriet** [comment]
[PHP Notice + Log.zip](https://github.com/nystudio107/seomatic/files/756956/PHP.Notice.Log.zip)
If you need any other information, please let me know.

**Comment by khalwat** [comment]
It looks to me that you have some kind of Twig error in this template meta: "products/_entry"

**Comment by MichaelLotriet** [comment]
I have "Template Meta" set up for Products and News. I went to see if it might be something I added in the Template Meta setup, but i got the exact same error when trying to edit the Template Meta for Products or News... So i went into the database to edit the Template Meta from there. In the SEO Description field i had " | slice('0, 160') " so I changed it to " | slice(0, 160) " and that fixed it. Sorry about this.
